test.js

```diff
@@ -32,7 +32,7 @@ test('preferred port unavailable', async t => {
 	t.not(port, desiredPort);
 });
 
-test('preferred port priviliged', async t => {
+test('preferred port privileged', async t => {
```

Comment:
@ oops, thanks for catching this typo @sindresorhus, and thanks for the quick merge!